create.cjs

```diff
@@ -615,33 +602,30 @@ async function createEsShimLikePackage(packageName, packageImplementation, isSta
         'module.exports = bound;',
         ''
       ].join('\n')
-    ),
-    compareAndWriteFile(
-      path.join(packagePath, 'package.json'),
-      `${JSON.stringify({
-        name: `@nolyfill/${packageName}`,
-        version: currentPackageJson.version,
-        repository: {
-          type: 'git',
-          url: 'https://github.com/SukkaW/nolyfill',
-          directory: `packages/generated/${packageName}`
-        },
-        main: './index.js',
-        license: 'MIT',
-        files: ['*.js'],
-        scripts: {},
-        dependencies: {
-          '@nolyfill/shared': 'workspace:*',
-          ...extraDependencies
-        },
-        engines: {
-          node: minimumNodeVersion
-        }
-      }, null, 2)}\n`
-    )
-  ]);
+    },
+    packageJson: {
+      name: `@nolyfill/${packageName}`,
+      version: currentPackageJson.version,
+      repository: {
+        type: 'git',
+        url: 'https://github.com/SukkaW/nolyfill',
+        directory: `packages/generated/${packageName}`
+      },
+      main: './index.js',
+      license: 'MIT',
+      files: ['*.js'],
+      scripts: {},
+      dependencies: {
+        '@nolyfill/shared': 'workspace:*',
+        ...extraDependencies
+      },
+      engines: {
+        node: minimumNodeVersion
+      }
+    }
+  };
 
-  console.log(`[${packageName}] created`);
+  await writePackage(pkg);
```

Comment:
```suggestion
  return writePackage(pkg);
```

Since `createEsShimLikePackage` is going to return a promise anyway, let's just return `writePackage(pkg)` here.